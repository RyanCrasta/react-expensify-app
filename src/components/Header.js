import {BrowserRouter, Route, Switch, Link, NavLink} from 'react-router-dom';
import React from 'react';

const Header = () => {
   
    
    return(
        <header>
            <h1>EXPENSIFY</h1>
            <NavLink  to='/' activeClassName='is-active' exact={true}>HOME   </NavLink>
            <NavLink  to='/create' activeClassName='is-active'>CREATE         </NavLink>
            <NavLink  to='/help' activeClassName='is-active'>HELP           </NavLink>
        </header>
    )
}

export {Header as default};