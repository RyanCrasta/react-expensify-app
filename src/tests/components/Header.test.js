// react-test-renderer
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import Header from '../../components/Header';

test('should render header correctly', () => {

    const wrapper = shallow(<Header />);

    //expect(wrapper.find('h1').text()).toBe('EXPENsify');
    expect(toJSON(wrapper)).toMatchSnapshot();



    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
    //console.log(renderer.getRenderOutput());
})

             