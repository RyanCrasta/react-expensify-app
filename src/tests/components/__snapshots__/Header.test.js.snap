// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header>
  <h1>
    EXPENsify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    HOME   
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    CREATE         
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    HELP           
  </NavLink>
</header>
`;
